spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/TaskCommands.java

```diff
@@ -255,13 +257,27 @@ public String stop(
 		return message;
 	}
 
+	@ShellMethod(key = ID_TYPE_LOG, value = "Retrieve task execution log using external or internal execution id")
+	public String retrieveExternalTaskExecutionLog(
```

Comment:
**[TinyNit]** The name of the method does not match the command desc nor impl. Although its an internal detail and not exposed to user.